core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -113,17 +106,15 @@ export class QuestionBackendApiService {
   }
 
   private _fetchQuestionSummaries(
-      skillIds: string[], cursor: string,
+      skillId: string, cursor: string,
       successCallback: (value: QuestionSummariesResponse) => void,
       errorCallback: (reason: string) => void): void|boolean {
-    if (!this.isListOfStrings(skillIds)) {
-      errorCallback('Skill ids should be a list of strings');
-      return false;
-    }
+    const skillIds = [skillId];
+
     var questionsDataUrl = this.urlInterpolationService.interpolateUrl(
       QuestionDomainConstants.QUESTIONS_LIST_URL_TEMPLATE, {
         comma_separated_skill_ids: skillIds.join(','),
-        cursor: cursor ? cursor : ''
```

Comment:
Why this change?

Reply:
@kevintab95 I think it was for this: https://github.com/oppia/oppia/pull/10867#discussion_r499272632. But I don't see this change anymore, I think it got overwritten in a future commit.

Reply:
Ah, missed that. Thanks for explaining!